ui/app/rif/pages/addNewMulticryptoAddress/domain-add-new-network-page.css

```diff
@@ -0,0 +1,77 @@
+.custom-select{
+    float: left;
+}
+.custom-button-success {
+    cursor: pointer;
+    width: 130px;
+    height: 40px;
+    color: #38D238;
+    background-color: #B4EEB4;
+    border: 2px solid #38D238;
+    border-radius: 5px;
+    font-size: 1.5rem;
+    font-weight: bold;
+    position: relative;
+    transition: all 0.2s;
+}
+.custom-button-success:hover {
+    transition: all 0.2s;
+    background-color: #DEF7DE;
+}
+.custom-button-cancel {
+    cursor: pointer;
+    width: 130px;
+    height: 40px;
+    color: #E52525;
+    background-color: #F08080;
+    border: 2px solid #E52525;
+    border-radius: 5px;
+    font-size: 1.5rem;
+    font-weight: bold;
+    position: relative;
+    transition: all 0.2s;
+}
+.custom-button-cancel:hover {
+    transition: all 0.2s;
+    background-color: #F7BEBE;
+}
+.full-width {
+    width: 100%;
+    position: relative;
+}
+.add-new-multicrypto-title {
+    text-align: center;
+    font-size: 1.5rem;
+    color: #3399FF;
+}
+.add-new-multicrypto-select {
+    text-align: center;
+    width: 90%;
+    margin-left: 5%;
+    margin-top: 20px;
+}
+.add-new-multicrypto-input {
+    margin-top: 20px;
+    margin-bottom: 70px;
+}
+.add-new-multicrypto-input input{
+    width: 90%;
+    margin: 5%;
+    font-size: 1rem;
+    border-top: 0px;
```

Comment:
The pixel unit is not necessary, you should remove it

Reply:
All pixel units were removed, now its using rem